frontend/src/components/Session.js

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from 'react'
+import { useSelector } from "react-redux"
+import { Login } from './Login'
+import { Secret } from './Secret'
+import { Signup } from './Signup'
+
+
+export const Session = () => {
+    const signUp = useSelector(store => store.auth.signUp)
+    const loggedIn = useSelector(store => store.auth.loggedIn)
+    return (
+        <>
+            {loggedIn === false && signUp === false && <Login />}
+            {loggedIn === false && signUp === true && <Signup />}
+            {loggedIn && <Secret />}
+        </>
```

Comment:
This was really nicely structured.  The only thing I wish to see here is a use of `localStorage` to see if the browser remembers the currect accessToken in order to go straight to `Secret`